doc/data/messages/c/consider-using-enumerate/details.rst

```diff
@@ -0,0 +1 @@
+Emitted when code that iterates with `range` and `len` is encountered. Such code can be simplified by using the `enumerate` builtin.
```

Comment:
Isn't this already described in the code itself ? https://pylint.pycqa.org/en/latest/messages/convention/consider-using-enumerate.html

https://github.com/PyCQA/pylint/blob/main/pylint/checkers/refactoring/recommendation_checker.py#L21

> Emitted when code that iterates with range and len is encountered. Such code can be simplified by using the enumerate builtin.

Reply:
I missed that - removed the `details.rst`. Code example is from the Python docs(https://docs.python.org/3/library/functions.html#enumerate).